apps/generator/docs/file-templates.md

```diff
@@ -64,4 +74,57 @@ export default function({ asyncapi }) {
     <File name={`file2.html`}>Content</File>
   ]
 }
-```
\ No newline at end of file
+```
+
+### Example 2: Rendering files based on the AsyncAPI Schema
+
+In practice, to render the multiple files, that are generated from the data defined in your AsyncAPI, you'll iterate over the array of schemas and generate a file for each schema as shown in the example below:
+
+```js
+/*
+ * To render multiple files, it is enough to return an array of `File` components in the rendering component, like in following example.
+ */
+export default function({ asyncapi }) {
+  const schemas = asyncapi.allSchemas();
+  // schemas is an instance of the Map
+  return Array.from(schemas).map(([schemaName, schema]) => {
```

Comment:
schemas is not an array but a map, so like I suggest in my previous comment to the PR, better use forEach

and yeah, you need to manually test if file is working, as this way you would know something is wrong

during generation I get `Generator Error: object is not iterable (cannot read property Symbol(Symbol.iterator))`

btw, this example also needs `import { File, Text } from "@asyncapi/generator-react-sdk";`